### PR TITLE
f files

### PR DESCRIPTION
**Pull request by jtlap**

### CODE REVIEW

File: include/eve/function/factorial.hpp [hunk 1]
```diff
@@ -29,7 +29,7 @@ namespace eve
   //! ---
   //!
   //!  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~{.cpp}
-  //!  template< unsigned_value T > auto operator()( T n ) const noexcept;
+  //!  auto operator()( nsigned_value auto n ) const noexcept;
```
Comment (jfalcou) [inline include/eve/function/factorial.hpp:32]: `unsigned_value`

File: include/eve/function/factorial.hpp [hunk 2]
```diff
@@ -38,7 +38,7 @@ namespace eve
   //!
   //! **Return value**
   //!
-  //! The result element type is always double to try to avoid overflow
+  //! The result [element type](eve::element_type_t) is always double to try to avoid overflow
```
Comment (jfalcou) [inline include/eve/function/factorial.hpp:41]: `eve::element_type`, `element_type_t` is a subpart of the documentation of the former

**Review by jfalcou — CHANGES_REQUESTED** [review]

File: include/eve/function/factorial.hpp (hunk 2 above)
Comment (jtlap) [reply include/eve/function/factorial.hpp:41]: done

File: include/eve/function/factorial.hpp (hunk 1 above)
Comment (jtlap) [reply include/eve/function/factorial.hpp:32]: done

Comment (jtlap) [reply include/eve/function/factorial.hpp:32]: done

**Review by jfalcou — APPROVED** [review]

**Pull request merged**